src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -474,3 +525,31 @@ def test_init_raises_driver_errors_for_invalid_arguments(resource_name, channels
             # sessions, so we make a driver call on channels and ensure that errors
             session.channels[channels].output_function = nidcpower.OutputFunction.DC_VOLTAGE
     assert e.value.code == expected_error_code
+
+
+def test_repeated_capabilities_on_method_when_all_channels_are_specified(session):
+    '''Sessions should not error when specifying all channels by number.'''
+    assert session.channels['0'].output_enabled is True
+    session.channels['0'].output_enabled = False
+    session.channels['0-11'].reset()
+    assert session.channels['0'].output_enabled is True
+
+
+@pytest.mark.legacy_session_only
+def test_error_channel_name_not_allowed_in_obsolete_session(session):
+    with pytest.raises(nidcpower.Error) as e:
+        session.channels['0'].reset()
+    assert e.value.code == -1074118494  # NIDCPOWER_ERROR_CHANNEL_NAME_NOT_ALLOWED_IN_OBSOLETE_SESSION
+    assert e.value.description.find('The channel name string must represent all channels in the session because the session was not initialized with independent channels. To specify a subset of channels for this function, first initialize the session with independent channels.') != -1
+
+
+@pytest.mark.legacy_session_only
+def test_error_channel_name_not_allowed(session):
+    with pytest.raises(nidcpower.Error) as e:
+        session.channels['0'].instrument_model
+    assert e.value.code == -1074134971  # IVI_ERROR_CHANNEL_NAME_NOT_ALLOWED
+    assert e.value.description.find('The channel or repeated capability name is not allowed.') != -1
+
+
```

Comment:
Some other test cases we should have:
 - `instruments` rep cap
 - A test similar to `test_repeated_capabilities_with_initiate`, but with multi-instrument session
- A test similar to `test_repeated_capabilities_with_initiate`, but with FQCN as resource_name
 - A test similar to `test_repeated_capabilities_with_initiate`, but with channels rep cap equal to output of `get_channel_names()`
 - Session opened on `Dev1/0` and method called with channels rep cap equal to `1`

Also, verify all API methods that were updated to add the Channels rep cap are tested at least once.

Reply:
Added coverage in cb25392bc1430e3e03e2d741363eebfea78a4403 and 7696d83dbb5fc0e6b79e694bac38807e0830f8a1